ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml

```diff
@@ -36,6 +40,16 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-upgrade
+  cron: 0 0 * * *
```

Comment:
Note: the default cron setting should be something like @yearly to avoid interfering with the release-controller, the current setting should be used just for showing the stability of the job (as required by https://docs.ci.openshift.org/docs/architecture/release-gating/#show-the-job-is-stable)

Reply:
@andfasano So if we want to show that it is stable, shouldn't we run it @daily ?

Reply:
I think it's acceptable only during the staging phase (ie until the PR remains open)